mig-api/api.go

```diff
@@ -250,7 +238,10 @@ func authenticate(pass handler, adminRequired bool) handler {
 		if !ctx.Authentication.Enabled {
 			inv.Name = "authdisabled"
 			inv.ID = 0
-			inv.IsAdmin = true
+			inv.Permissions.DefaultSet()
+			inv.Permissions.ManifestSet()
+			inv.Permissions.LoaderSet()
+			inv.Permissions.AdminSet()
```

Comment:
Yeah good point, we could do something like inv.Permissions.FromMask(0xffffffffffffffff)